bower.json

```diff
@@ -11,17 +11,17 @@
     "angular-ui-router": "~0.2.15",
     "bootstrap": "~3.3.6",
     "angular-bootstrap": "~1.3.2",
-    "angular": "~1.5.0",
+    "angular": "~1.5.5",
```

Comment:
@rnugraha Actually I did not intend to make this change - do you think we should upgrade to 1.5.5 ? If not I will remove it.